### PR TITLE
Default tracker disable flag

### PR DESCRIPTION
**Pull request by DveMac**
- Update initialize options
- Add unit tests
- Update readme

### CODE REVIEW

**Comment by SimeonC** [comment]
Thanks for the quick PR, could you remove the editorconfig though. That should be done in a different PR - it’s a good idea but my preference is prettier which doesn’t require IDE integration.

Do you want to be added as a maintainer? You’ve proven you would be a good contributor.

**Comment by DveMac** [comment]
Updated. Happy to help maintain. But my time is a little limited, so not sure how much I can help.

**Comment by rvaidya** [comment]
Can _gaCommand be changed to something like this?  You get the best of both worlds then - with the current PR, you lose the default tracker completely if the option is false

```
function _gaCommand(trackerNames, ...args) {
  const command = args[0];
  if (typeof internalGa === 'function') {
    if (typeof command !== 'string') {
      warn('ga command must be a string');
      return;
    }

    if (_alwaysSendToDefaultTracker) internalGa(...args);
    if (Array.isArray(trackerNames)) {
      trackerNames.forEach((name) => {
        internalGa(...[`${name}.${command}`].concat(args.slice(1)));
      });
    }
    else if(!_alwaysSendToDefaultTracker) internalGa(...args);
  }
}
```

File: src/index.js [hunk 1]
```diff
@@ -41,7 +42,7 @@ function _gaCommand(trackerNames, ...args) {
       return;
     }
 
-    internalGa(...args);
+    if (_alwaysSendToDefaultTracker) internalGa(...args);
```
Comment (SimeonC) [inline src/index.js]: From your description of the parameter in the Readme this should be `if (_alwaysSendToDefaultTracker || !Array.isArray(trackerNames)) internalGa(...args);`.
I think @rvaidya is correct that this option should only take effect when trackerNames are not defined (like you put in the readme too).
I think with that change we should be good to go.

BTW I added you as a collaborator anyway, hopefully 2 of us with not enough time is enough to keep it maintained.

**Comment by DveMac** [comment]
@SimeonC @rvaidya Thanks for your input guys, I have updated accordingly.

In hindsight it would have been better to require _all_ trackers to be specified in my original PR. Maybe this change can be introduced in the next major version bump. e.g.  `ReactGA.event(args, ['default', 'tracker1'])`

**Comment by SimeonC** [comment]
Merged. That breaking solution does sound like a cleaner solution overall, but it definitely would have to be a major version change.

**Pull request merged**